app_description.txt

```diff
@@ -0,0 +1,49 @@
+Zingo! A Zcash Application
+
+Use zero knowledge cryptography to manage your transaction history in a private and mindful way.
```

Comment:
I don't want to bury the lead on the fact that Zingo has full wallet functionality. Maybe something like:
```suggestion
Use zero knowledge cryptography to transact, and to manage your transaction history, in a private and mindful way.
```